apps/backoffice-v2/src/pages/Entities/components/CaseGeneration/CaseGeneration.tsx

```diff
@@ -0,0 +1,82 @@
+import { Button } from '@/common/components/atoms/Button/Button';
+import { SheetContent, SheetTrigger } from '@/common/components/atoms/Sheet';
+import { Sheet } from '@/common/components/atoms/Sheet/Sheet';
+import { CaseGenerationForm } from '@/pages/Entities/components/CaseGeneration/components/CaseGenerationForm';
+import { withCaseGeneration } from '@/pages/Entities/components/CaseGeneration/context/case-generation-context/hocs/withCaseGeneration';
+import { useCaseGenerationContext } from '@/pages/Entities/components/CaseGeneration/context/case-generation-context/hooks/useCaseGenerationContext';
+import { useCaseGenerationWorkflowDefinition } from '@/pages/Entities/components/CaseGeneration/hooks/useCaseGenerationWorkflowDefinition';
+import { PlusIcon } from '@radix-ui/react-icons';
+
+export const CaseGeneration = withCaseGeneration(() => {
```

Comment:
Changes thus far use terminology around "create" and "creation" let's keep that consistent.
```suggestion
export const CaseCreation = withCaseCreation(() => {
```